MonstieWash/Assets/Scripts/MonsterAi/MonsterBrain.cs

```diff
@@ -296,5 +306,25 @@ private float FloatTowardsTarget(float value, float change, float target)
         return value;
     }
 
+    /// <summary>
+    /// Returns the ID (as an int) of the Moodtype with the highest value.
+    /// </summary>
+    public string GetHighestMood()
+    {
+        float highestVal = float.MinValue;
+        int highestValID = 0;
+        foreach(var mood in activeMoods)
+        {
+            if (mood.Value > highestVal) 
+            {
+                highestValID = mood.Key;
+                highestVal = mood.Value;
+            }
+        }
+
+        string highestMoodName = ReadMood(highestValID);
+
+        return highestMoodName;
+    }
```

Comment:
This should have a return value comment and also don't forget to be using var wherever possible. Also try to remember to separate blocks of code (literally just the first block lol).
```suggestion
    /// <summary>
    /// Returns the ID (as an int) of the Moodtype with the highest value.
    /// </summary>
    /// <returns>The name of the highest mood.</returns>
    public string GetHighestMood()
    {
        var highestVal = float.MinValue;
        var highestValID = 0;
        
        foreach(var mood in activeMoods)
        {
            if (mood.Value > highestVal) 
            {
                highestValID = mood.Key;
                highestVal = mood.Value;
            }
        }

        var highestMoodName = ReadMood(highestValID);

        return highestMoodName;
    }
```